modules/ank/arrow-ank/build.gradle

```diff
@@ -2,11 +2,13 @@ apply plugin: 'kotlin'
 apply plugin: 'kotlin-kapt'
 
 dependencies {
-    compile project(":arrow-core")
     compile project(":arrow-fx")
 
-    compile "org.jetbrains.kotlin:kotlin-compiler:$ankKotlinVersion"
-    compile "org.jetbrains.kotlin:kotlin-script-util:$ankKotlinVersion"
+    runtime "org.jetbrains.kotlin:kotlin-compiler:$kotlinVersion"
```

Comment:
These dependencies come from https://github.com/JetBrains/kotlin/blob/01a05a5495049668c0255c9b6f42adb17245a32b/libraries/examples/kotlin-jsr223-local-example/build.gradle.kts#L16
in order to use the ScriptEngine. That was one of the reasons it didn't work in #1878